src/main.ts

```diff
@@ -0,0 +1,30 @@
+import { Config, OverrideStandard } from './types';
+import generateDiff from './generateDiff';
+import { standard } from './standard';
+import categorizeChanges from './categorizeChanges';
+import AsyncAPIDiff from './asyncapidiff';
+import { mergeStandard } from './mergeStandard';
+
+/**
+ * Generates diff between two AsyncAPI documents
+ * @param firstDocument The parsed AsycnAPI document
+ * @param secondDocument The parsed AsycnAPI document
+ * @param {Config} config Configuration options
```

Comment:
we need more jsdoc details on the configuration options, what options are basically available. Would be good to even include an example like https://github.com/asyncapi/generator/blob/master/lib/generator.js#L144-L165

Reply:
like what this `override` means after all, what is the purpose. What is `add`, what is `remove` and what is `edit`

Reply:
yeah, I'm going to add this now. I was working on document generation and JSDoc just gave me `Config`. I thought it will give things inside them as well.